app/views/pages/adopt.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="stats">
+  <ul>
+    <li>10<span>Rats available</span></li>
```

Comment:
Are these numbers placeholders? What do you think of replacing the numbers with the word placeholder so we don't forget to change them later?